client/src/hooks/reducers/feedReducers.js

```diff
@@ -34,3 +35,12 @@ export const optionsReducer = (oldState, action) => {
       return oldState;
   }
 };
+
+export const postsReducer = (oldState, action) => {
```

Comment:
I think the `SET_VALUE` in `feedReducer` does the same thing? But I'm not sure what the best practices are.

Reply:
It's separated because of the API integration that needs to be done to have separated objects, but they can be integrated into the same reducer if you want.

Reply:
Ah ok good to know, let's keep like this then.